Visualization/bokeh_holoviews_datashader.ipynb

```diff
@@ -538,7 +538,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "print('The value at position (%.3f, %.3f) is %.3f'%(posxy.x, posxy.y, img[int(posxy.x), int(posxy.y)]))"
+    "print('The value at position (%.3f, %.3f) is %.3f'%(posxy.x, posxy.y, image.image.array[int(posxy.x), int(posxy.y)]))"
```

Comment:
In order to get the correct value at the selected position, I had to edit this line to be
`print('The value at position (%.3f, %.3f) is %.3f'%(posxy.x, posxy.y, image.image.array[-int(posxy.y), int(posxy.x)]))`
which seems to be a definition of the origin in the array

Reply:
Good catch.  The origin in these plots is the upper left, I believe.